docs/framework/get-started/the-net-framework-and-out-of-band-releases.md

```diff
@@ -47,8 +47,8 @@ OOB releases for core common language runtime (CLR) components are delivered thr
 
  To find both prerelease and stable packages, choose **Include Prerelease** in the NuGet Package Manager.
 
- If you want to be notified of stable package releases, subscribe to [the .NET Framework feed](https://nuget.org/api/v2/curated-feeds/dotnetframework/Packages/).
+ If you want to be notified of stable package releases, subscribe to [the .NET Framework feed](https://www.nuget.org/api/v2/curated-feeds/microsoftdotnet/Packages/).
```

Comment:
Tip: Another link is at docs/docs/framework/whats-new/index.md#L31.

Reply:
@NextTurn thanks!